internal/collectors/container_runtime.go

```diff
@@ -0,0 +1,76 @@
+package collectors
+
+import (
+	"os"
+	"regexp"
+
+	"github.com/SUSE/connect-ng/internal/util"
+	"github.com/genuinetools/bpfd/proc"
```

Comment:
While I see no direct problem with this package. It appears pretty unmaintained and stale. Besides its a full blown syscall tracing application. Do we really need this to make the detection happening?

I personally would rather copy the code from: https://github.com/genuinetools/bpfd/blob/master/proc/proc.go#L96 or kaniko the equivalent.

I see problems appear with this package, not on a technical level but rather organizational level.

What do you think?

Reply:
I will simply port the code from https://github.com/GoogleContainerTools/kaniko since it has the latest updates and is fairly up-to-date.